.eslintrc

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
please remove this file

we will port the changes to new rule in a separate pr